spec/services/calculators/immediately_calculator_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe Calculators::ImmediatelyCalculator do
+  subject { described_class.new(disclosure_check) }
+
+  context '#expiry_date' do
+    let(:disclosure_check) { build(:disclosure_check,
+                                   known_date: known_date,
+                                   conviction_length: conviction_length,
+                                   conviction_length_type: conviction_length_type) }
+
+    let(:known_date) { Date.new(2018, 10, 31) }
+    let(:conviction_length) { 10 }
+
+    context 'conviction length in months' do
+      let(:conviction_length_type) { 'months' }
+
+      it 'returns expiry date 10 months advance' do
```

Comment:
I think these tests doesn't correspond with this calculator. Probably copy paste.